src/components/radio/radio-button.scss

```diff
@@ -0,0 +1,139 @@
+@import "default-theme";
+
+$radio-width: 20px !default;
+$radio-height: $radio-width !default;
+
+// Top-level container for entire button.
+md-radio-button {
+  display: inline-block;
+  cursor: pointer;
+  padding: 8px;
```

Comment:
Possible, though since we don't have any systematic guidance of that kind across components, my imagination is that users should just override the styles using their own cascaded styles.